integration-tests/scripts/start-integration-container.sh

```
@@ -239,6 +239,34 @@ for gateway_instance in "api-sheriff-cookie:19002" "api-sheriff-cookie-2:19003"
+# loop's probe URL is hard-coded to https://, and this instance's whole purpose is that its
+# management port is PLAIN HTTP (it selects the key-less `plain-management` TLS bucket via
+# QUARKUS_MANAGEMENT_TLS_CONFIGURATION_NAME). Adding it to the loop would probe https:// against a
+# plain listener and hang for the full 30 seconds. Keep the scheme here as http:// — if this probe
+# ever needs -k, the opt-out has silently stopped working and that is the bug, not the probe.
+echo "⏳ Waiting for the api-sheriff-plain-mgmt gateway instance to be ready..."
+for i in {1..30}; do
+    if curl -sf http://localhost:19005/q/health/live > /dev/null 2>&1; then
+        echo "✅ api-sheriff-plain-mgmt gateway instance is ready (management on PLAIN HTTP, deliberately)!"
+        break
+    fi
+    if [ $i -eq 30 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)